Modules/System/Azure Queues/src/AzureQueueSetup.Table.al

```diff
@@ -0,0 +1,32 @@
+/// <summary>
+/// Holds information about storage accounts and corresponding shared access signatures for handling queues.
+/// </summary>
+table 50100 "Azure Queue Setup"
+{
+    Caption = 'Azure Queue Setup';
+    DataClassification = SystemMetadata;
+
+    fields
+    {
+        field(1; "Storage Account Name"; Text[100])
+        {
+            Caption = 'Azure Storage Account Name';
+            Description = 'Name of the Azure Storage Account.';
+        }
+
+        field(2; "SAS Key"; Text[255])
```

Comment:
Is storing a SAS key in a table secure enough? Is this something that we should consider storing in the isolated storage?

Reply:
Personally, I would prefer a more generic way as I already commented some time ago.
Is it possible to reuse the Storage Authentication or can this be redesigned to accept an IAuthentication Interface instead of using a setup table.
Then we would be able to setup multile authentications for different purposes in our own tables with Isolated Storage